global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "3.0.23"
   },
   "sdk": {
-    "version": "6.0.100",
+    "version": "6.0.403",
```

Comment:
It doesn't look like we've added net6.0 to the TFM for `Azure.Core.Tests.csproj` yet. I think we should add it

Reply:
.net6.0 is added to all test projects: https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L123-L125

Reply:
Oh, right 🤦 😄